app/build.gradle

```diff
@@ -19,6 +19,7 @@ android {
 
     buildFeatures {
         viewBinding true
+        dataBinding true
```

Comment:
to you need both dataBinding and viewBinding? Looks like you can remove viewBinding